tools/icu/icu_small.json

```diff
@@ -25,7 +25,7 @@
     "zone": "locales",
     "converters": "none",
     "stringprep": "locales",
-    "translit": "none",
+    "translit": "locales",
```

Comment:
yeah - "locales" may not be quite right here in fact. It's more complex which is why I'd like a separate issue tracking this (translit enablement for v8)  Better would be to actually remove the `translit` line altogether.